modules/infrastructure/api-redis/src/main/resources/application-redis.yml

```
@@ -0,0 +1,5 @@
+spring:
+  data:
+    redis:
+      host: ${API_REDIS_HOST}
+      port: ${API_REDIS_PORT}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)